src/react/components/_finances/TransactionsCard/Filters/BillingItemSearch.scss

```diff
@@ -18,7 +18,7 @@
   background-position-x: 10px;
   background-position-y: center;
 
-  font-size: 12px;
+  font-size: $base-font-size;
   padding-left: 25px;
   padding-right: 10px;
   height: 32px;
```

Comment:
@developish just wondering if we should/could make the height 30px or 35px to align better with the 5px increments we are trying to use?

Reply:
I think I should bring that up with Kaylin when we put this in smoke testing. Currently if I set that value to 30px, it doesn't actually take because of the font-size and line-height.